modules/text2vec-weaviate/clients/weaviate_embed.go

```diff
@@ -78,15 +79,15 @@ func (v *vectorizer) Vectorize(ctx context.Context, input []string,
 	cfg moduletools.ClassConfig,
 ) (*modulecomponents.VectorizationResult, *modulecomponents.RateLimits, int, error) {
 	config := v.getVectorizationConfig(cfg)
-	res, err := v.vectorize(ctx, input, config.Model, config.Truncate, config.BaseURL, false, config)
-	return res, nil, 0, err
+	return v.vectorize(ctx, input, config.Model, config.Truncate, config.BaseURL, false, config)
```

Comment:
could you keep returning the 0 here for now? (Unless there is a token limit?)

I haven't wired up the code path that returns token usage to the user yet, but want to have everything in place :)